src/content/docs/en/concepts/islands.mdx

```diff
@@ -1,12 +1,12 @@
 ---
-title: Astro Islands
-description: "Astro Islands (aka Component Islands) are a pattern of web architecture pioneered by Astro. “Islands architecture” was first coined by Etsy's frontend architect Katie Sylor-Miller in 2019, and expanded on by Preact creator Jason Miller."
+title: Island Architecture
```

Comment:
Any reason we are moving from Islands Architecture to Island Architecture? I generally feel the former makes more sense since the idea is that you'd generally have many islands in a page, vs. one. Silly, I know, but there's a good amount of blog posts, etc. using the former and it doesn't feel like a good wording break change to make.

Reply:
This might have been a mistake, or unintentional! We should probably revert to the phrase we've always used. We can change it back later if Fred really did intend this change.